packages/components/src/components/select/select/select-styles.ts

```diff
@@ -37,6 +37,14 @@ import { getPopoverResetJssStyle } from '../../../styles/popover-reset-styles';
 import { OPTION_HEIGHT } from '../../../styles/option-styles';
 import { INTERNAL_SELECT_SLOT } from './select-utils';
 
+const cssVarBackgroundColor = '--p-select-background-color';
+const cssVarTextColor = '--p-select-text-color';
+const cssVarBorderColor = '--p-select-border-color';
+const cssVarIconFilter = '--p-select-icon-filter';
+
+const cssVarBackgroundColorFocus = '--p-select-focus-background-color';
+const cssVarBorderColorFocus = '--p-select-focus-border-color';
+
```

Comment:
Do we have a naming convention of CSS Vars and corresponding CONST? Is it expected that they are different? "BorderColorFocus" vs. "focus-border-color"

Reply:
`select:focus {background-color: #123}` => `--p-select-focus-background-color`